eng/Version.Details.xml

```diff
@@ -7,5 +7,9 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>16d4350184cf362cd59807b589d1c93803025abc</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.DotNet.ApiCompat" Version="5.0.0-beta.20162.4">
+      <Uri>https://github.com/dotnet/arcade</Uri>
+      <Sha />
```

Comment:
Oops - meant to remove this. I don't think we want to put it in Version.Details.xml at all since there will be no dependency flow from jbevain/cecil. That seems to be in line with how the SDK handles Newtonsoft.Json.

Reply:
It's empty because that's how `darc add-dependency` adds it. I think when dependency flow updates it will add the Sha.